python/tank/authentication/ui/login_dialog.py

```diff
@@ -79,7 +77,14 @@ def setupUi(self, LoginDialog):
 "QComboBox::disabled {\n"
 "    background-color: rgb(60, 60, 60);\n"
 "    color: rgb(160, 160, 160);\n"
-"}")
+"}\n"
+"\n"
+"QPushButton.main\n"
+"{\n"
+"    background-color: rgb(35, 165, 225);\n"
+"    color: rgb(248, 248, 248);\n"
+"}\n"
+"")
```

Comment:
I'm wondering if we can replace this big string with a triple quote notation string to stop adding explicit line breaks.

Reply:
Me too but it's an autogenerated file from an XML UI file (QT Designer).